ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/web3j/JsonRpcErrorCodes.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Consensys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.ethereum.executionclient.web3j;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public final class JsonRpcErrorCodes {
+  public static final int PARSE_ERROR = -32700;
```

Comment:
I'd convert this to an Enum with `errorCode` and `description` fields, so you can avoid this static fields\initialization